### PR TITLE
On workspace creation - open IDE by default, allow RAM < 0.5

### PR DESCRIPTION
**Pull request by ashumilova**
Signed-off-by: Anna Shumilova <ashumilo@redhat.com>

### What does this PR do?
1. Plugins list on workspace creation is visible only if Che7 stack is selected
2. Add ability to set the RAM less then 512MB
3. Filter all stacks for worskpace creation by supported infrastructure (unsupported ones won't be shown now in the "All" tab as well)
4. "Create & Open" buttons - by default perform workspace creation and opening in IDE, the popup with choices to open IDE or Edit won't be shown; at the top - there is button to "Create & Proceed Editing"
5. Selenium test fixes due to changes in workspace creation flow

ezgif com-video-to-gif 4
ezgif com-video-to-gif 5

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/11788

### CODE REVIEW

**Comment by ashumilova** [comment]
Need to check all the tests.

**Comment by ashumilova** [comment]
ci-test

**Review by akurinnoy — APPROVED** [review]

**Comment by ashumilova** [comment]
@slemeur can you, please, verify the workspace creation flow is fine?

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1266/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1266//Selenium_20tests_20report/)
docker image: eclipseche/che-server:12107
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Review by l0rd — APPROVED** [review]
LGTM!

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/AddOrImportProjectFormTest.java [hunk 1]
```diff
@@ -428,25 +428,18 @@ public void checkCreatingProject() throws Exception {
     addOrImportForm.clickOnAddButton();
     addOrImportForm.waitProjectTabAppearance(SPRING_SAMPLE_NAME);
 
-    // check closing of "Workspace Is Created" dialog window
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/AddOrImportProjectFormTest.java:431]: It's interesting to know why this verification had been removed?

**Review by dmytro-ndp — DISMISSED** [review]
Changes includes removal and fixing the methods in NewWorkspace.java selenium page object, which may be used in E2E tests in downstream projects. We need to verify if other changes don't course regression in downstream projects as well.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/AddOrImportProjectFormTest.java (hunk 1 above)
Comment (ashumilova) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/AddOrImportProjectFormTest.java:431]: The workspace creation flow is changing - see the issue description and comments.

**Review by olexii4 — APPROVED** [review]

**Comment by eivantsov** [comment]
@dmytro-ndp what downstream projects?

**Comment by SkorikSergey** [comment]
There are still selenium tests to update according to this issues changes. I will do it today.

**Comment by SkorikSergey** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1276/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1276//Selenium_20tests_20report/)
docker image: eclipseche/che-server:12107
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by SkorikSergey** [comment]
Selenium tests execution on Eclipse Che Multiuser on OCP (https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1276/) doesn't show any regression against this Pull Request.

**Review by SkorikSergey — APPROVED** [review]

**Review by benoitf — APPROVED** [review]

**Comment by ashumilova** [comment]
squashed 3 commits into one

**Comment by sleshchenko** [comment]
`"Create & Open"` makes things much clearer than was before with `Create` with two behaviors!!! <3

**Pull request merged**